templates/cluster-template-rosa.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: "${CLUSTER_NAME}"
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks: ["192.168.0.0/16"]
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta2
+    kind: ROSACluster
+    name: "${CLUSTER_NAME}"
+  controlPlaneRef:
+    apiVersion: controlplane.cluster.x-k8s.io/v1beta2
+    kind: ROSAControlPlane
+    name: "${CLUSTER_NAME}-control-plane"
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta2
+kind: ROSACluster
+metadata:
+  name: "${CLUSTER_NAME}"
+spec: {}
+---
+apiVersion: controlplane.cluster.x-k8s.io/v1beta2
+kind: ROSAControlPlane
+metadata:
+  name: "${CLUSTER_NAME}-control-plane"
+spec:
+  version: "${OPENSHIFT_VERSION}"
+  region: "${AWS_REGION}"
+  accountID: "${AWS_ACCOUNT_ID}"
+  creatorARN: "${AWS_CREATOR_ARN}"
+  machineCIDR: "10.0.0.0/16"
+  rolesRef:
+    ingressARN: "arn:aws:iam::${AWS_ACCOUNT_ID}:role/${OPERATER_ROLES_PREFIX}-ingress-operator-cloud-credentials"
```

Comment:
ingress, registry, storageand. network should be `${OPERATER_ROLES_PREFIX}-openshift-...`